js/iteration.js

```diff
@@ -7,39 +7,42 @@
 export const IterationAPI = superclass =>
     class extends superclass {
         /**
-         * Returns all iterations in the registry.
+         * Returns all iterations for a given project.
          *
          * @memberof IterationAPI
+         * @param {String} projectName The name of the project.
          * @returns {Promise} The iteration result list.
          */
-        async listIterations() {
-            const url = this.baseUrl + "iterations";
+        async listIterations(projectName) {
+            const url = this.baseUrl + `projects/${projectName}/iterations`;
             const contents = await this.get(url);
             return contents;
         }
 
         /**
-         * Returns the iteration with the provided id.
+         * Returns the iteration with the provided id within a given project.
          *
          * @memberof IterationAPI
+         * @param {String} projectName The name of the project.
          * @param {String} id The identifier of the iteration.
          * @returns {Promise} The iteration requested.
          */
-        async getIteration(id) {
-            const url = this.baseUrl + `iterations/${id}`;
+        async getIteration(projectName, id) {

```

Comment:
NAming for this method should be `iterationProject`

https://github.com/hivesolutions/github_api/blob/master/src/github/repo.py